tutorials/pl/basic-snap-usage-pl/basic-snap-usage.md

```diff
@@ -0,0 +1,329 @@
+
+### Usuwanie snapów
+
+Usunięcia snapa wymaga tylko jednego polecenia, wszystko co musisz zrobić to `snap remove <snap name>`. W naszym przypadku jest to:
+
+
+```bash
+$ sudo snap remove hello-world
+
+hello-world removed
+```
+
+Łatwo i przejrzyście, nic po nim nie pozostało! Kod aplikacji, jej zależności i dane użytkownika są usuwane. Jeżeli snap zawiera usługę, zostanie ona wyłączona i usunięta.
+
+
+## Korzystanie z wersji i kanałów
+Duration: 2:00
+
+Programiści mogą wydawać wersje stabilne, kandydujące do stabilnej, beta i niestabilne snapów w tym samym czasie, aby zaangażować społeczność, która chciałaby testować nadchodzące zmiany. Możesz wybrać, z którego kanału będziesz korzystać.
+
+Domyślnie, snapy są instalowane z kanału `stable`. Zgodnie z konwencją, programiści korzystają z kanału `candidate` aby dostarczać nadchodzącą wersję stabilną, pozwalając na przetestowanie jej. Kanał `beta` zawiera nieukończone, ale znaczące zmiany, kanał `edge` jest zwykle używany dla regularnych lub dziennych kompilacji, które pomyślnie skompilowano.
+
+### Zmiana kanału zainstalowanej aplikacji
+
+
+```bash
+$ sudo snap refresh hello --channel=beta
+
+hello (beta) 2.10.1 from 'canonical' refreshed
+```
+
+Możesz teraz uruchomić wersję beta pakietu i sprawdzić, czy działa tak samo:
+
+
+```bash
+$ hello
+Hello, snap padawan!
+```
+
+`snapd` podąża teraz za kanałem `beta` pakietu `hello` i pobierze każde aktualizacje udostępniane za jegi pośrednictwem.
+
+
+```bash
+Możesz też zainstalować pakiet bezpośrednio z pakietu beta poleceniem
+`$ sudo snap install hello --beta`
+```
+
+### Odwróć zmiany, jeśli coś poszło nie tak!
+
+Jedną z możliwości `snapd` jest funkcja pozwalająca na powrót do poprzedniej wersji aplikacji (wraz z danymi powiązanymi ze snapem) z dowolnego powodu:
+
+
+```bash
+$ sudo snap revert hello
+
+hello reverted to 2.10
+$ hello
+Hello, world!
+```
+
+Dzięki wbudowanemu systemowi przywracania zmian, możesz bez obaw dokonywać aktualizacji wiedząc, że będziesz mieć możliwość powrotu do poprzedniego stanu!
+
+
+## To już wszystko!
+Duration: 1:00
+
+### Proste, prawda?
+
+Gratulujemy! Udało Ci się!
+
+Umiesz już szukać snapów w sklepie, instalować i aktualizować je, zmieniać kanały i wiele więcej. Polecenie `snap` zostało zaprojektowane, aby być tak proste i zapamiętywalne jak to możliwe. Po kilku użyciach, korzystanie z niego będzie dla Ciebie banalne.
+
+### Następne kroki
+
+  - Zapoznaj się w bardziej zaawansowane funkcje i możliwości snapów czytając poradnik „[Advanced snap usage]”.
+  - Zaskocz się tym, jak łatwo możesz utworzyć własnego snapa po przeczytaniu „[Creating your first snap]”.
+  - Dołącz do społeczności snapcraft.io na [forum snapcrafta].
+
+### Do przeczytania później
+
+  - [Dokumentacja snapcraft.io] jest dobrym miejscem, w którym przeczytasz pełną dokumentację snapa i snapcrafta.
+  - Dowiedz się, jak [skontaktować się z nami i szerszą społecznością].
+
+
+
+
+
+[snap]: http://snapcraft.io/
```

Comment:
This list needs gardening based on my suggestions / remarks above.

Reply:
Corrected almost everything. The original also needs to be updated…